src/renderer/src/shared/styles/font.css.ts

```
@@ -0,0 +1,37 @@
+import { css } from 'styled-components'
+
+const fontGenerator = (
+  weight: number,
+  size: number,
+  lineHeight: number,
+  letterSpacing: number
+) => css`
```

ESLint

[error] Missing return type on function.

(@typescript-eslint/explicit-function-return-type)